core/google/cloud/iterator.py

```diff
@@ -242,7 +242,8 @@ def _page_iter(self, increment):
                           results per page while an items iterator will want
                           to increment per item.
 
-        Yields :class:`Page` instances.
+        :rtype: :class:`Page`
+        :returns: pages
```

Comment:
I'd be happier just converting to a google-style docstring here (since it actually supports `Yields`). WDYT?